galaxy_ng/tests/integration/api/test_iqe_rbac.py

```diff
@@ -31,7 +31,6 @@
 class TestRBAC:
 
     @pytest.mark.iqe_rbac_test
-    @pytest.mark.standalone_only
```

Comment:
The rbac tests are part of a different github action. Since we're running the main tests as `deployment_standalone or all` these tests would get picked up under then main job.